wordpress-mobile/WordPress-Android
# Send Android lint reports to GitHub Code Scanning

### Description

This PR introduces integration with GitHub Code Scanning feature for Android lint.

Internal project reference: paaHJt-7ay-p2

## To Test:

Check: https://github.com/wordpress-mobile/WordPress-Android/security/code-scanning?page=1&query=is%3Aopen+branch%3Atrunk+pr%3A21207 and verify there are issues reported by Android lint.
